.github/workflows/vulnerability_scanning.yml

```diff
@@ -8,23 +8,32 @@ on:
         ## use github hosted runner by default
         default: "ubuntu-latest"
         required: false
+      project:
+        required: true
+        type: string
+      component:
+        required: false
+        type: string
+    secrets:
+      MINIO_CN_ACCESS_KEY_ID:
+        required: true
+      MINIO_CN_SECRET_ACCESS_KEY:
+        required: true
```

Comment:
they are not required, if `docker_image_tar` is empty, then you can run repo scanning mode by default

Reply:
@victorchen2022 ^